airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -5,34 +5,24 @@ tests:
   connection:
     - config_path: "secrets/config.json"
       status: "succeed"
-    - config_path: "secrets/connected_account_config.json"
-      status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
       status: "failed"
   discovery:
     - config_path: "secrets/config.json"
-    - config_path: "secrets/connected_account_config.json"
   basic_read:
+    # Test for subscriptions only with respect to subscription status in ["active","canceled"]
+    # If this test fails for some reason, please check the expected_subscriptions_statuses_records.json for valid subset of data.
     - config_path: "secrets/config.json"
-#     Reading a invoice_line_items stream takes too long and falls on timeout.
```

Comment:
why did you remove these test cases?

Reply:
on the stage of acceptance tests they are failing, did you know that those tests are never get triggered, because previously this connector was on the `standard-source-test` not on `source-acceptence-test`, and as far as I checked the `acceptence-test-config.yaml` is only triggered with `source-acceptence-tests`.

I'll try to bring them back, and test again.

Reply:
The next schemas are not valid at this point and these should be edited corresponding to the real data types in the output:
`balance-transactions, bank_accounts, charges, customer_balance_transactions, customers, disputes, events, invoice_items, invoices, payouts, products, transfers`

Reply:
thanks!

Reply:
Done. Tests are fixed, the timeout value of 3600 is set, because from time to time tests are passing too long.